strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenProxyRepositoryProviderTestIT.java

```diff
@@ -168,15 +152,35 @@ public void shouldBeAbleToProvideFilesFromOracleMavenRepoWithHttpsAndAuthenticat
     }
     */
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+            ArtifactManagementTestExecutionListener.class })
```

Comment:
You are not creating artifacts, hence you don't need `ArtifactManagementTestExecutionListener.class`.